en/docs/setup/deployment-guide.md

```diff
@@ -42,7 +42,9 @@ necessary system requirements and a compatible environment.
 </tbody>
 </table>
 !!! note
-    For more information on prerequisites, see [Installation Prerequisites](../../setup/installation-prerequisites)
+    We do not recommend running WSO2 Identity Server on ARM architecture. However, official support for this is being considered for future releases.
```

Comment:
Is this `Arm architecture`?
@sanjulamadurapperuma

Reply:
Yes, that's correct